website/docs/r/sentinel_alert_rule_scheduled.html.markdown

```diff
@@ -81,6 +83,28 @@ The following arguments are supported:
 
 * `trigger_threshold` - (Optional) The baseline number of query results generated, combined with `trigger_operator`, setting alert threshold of this Sentinel Scheduled Alert Rule.
 
+---
+
+A `grouping_configuration` block supports the following:
```

Comment:
shouldn't this be under the incident configuration block desription?

Reply:
The order here follows alphabetic order (as implemented in the scaffolding tool by Tom). The referencing verb is properly handled - if you look at the statement where the `grouping` is referenced in `incident_configuration` block, it uses statement:

> `grouping` - (Optional) A `grouping` block as defined above.

Reply:
As much as that may be the case, wouldn't it make more sense to have the sub blocks documented after the block they are contained in? otherwise the user will scroll down and up continually and there is no structure to the documentation?

Reply:
Agreed, I will update it.